README.md

```diff
@@ -20,21 +20,22 @@ In order to connect readers familiar with the former name of the specification i
 
-* *The OpenAPI Specification (OAS) provides an open governance model, as directed by the Open API Initiative's charter. The specification, although managed by the OAI, is not the same as the initiative itself.*
+* *The OpenAPI Specification (OAS) provides an open governance model, as directed by the OpenAPI Initiative's charter. The specification, although managed by the OAI, is not the same as the initiative itself.*
 
 ## References to Teams in the Project
 
-As per the charter, the Open API Initiative (OAI) provides an open source, technical community, within which industry participants may contribute to building a vendor-neutral, portable, and open specification for providing technical metadata for REST APIs — the Open API Specification (OAS). The following named groups may be properly referred to as:
+As per the charter, the OpenAPI Initiative (OAI) provides an open source, technical community, within which industry participants may contribute to building a vendor-neutral, portable, and open specification for providing technical metadata for REST APIs — the OpenAPI Specification (OAS). The following named groups may be properly referred to as:
 
 * **Business Governance Board** ("BGB", second reference)
 
-    * Comprised of official corporate entities (companies, schools, non-profits, etc.) who have signed the official project membership agreement to be members of the Open API Initiative and are bound by the laws governed by the project’s charter. [[Read more here](https://www.openapis.org/participate/how-to-contribute/governance#BGB)]
+    * Comprised of official corporate entities (companies, schools, non-profits, etc.) who have signed the official project membership agreement to be members of the OpenAPI Initiative and are bound by the laws governed by the project’s charter. [[Read more here](https://www.openapis.org/participate/how-to-contribute/governance#BGB)]
 
-* **Technical Developer Community** ("TDC", second reference)
+* **Technical Steering Committee** ("TSC", second reference)
 
-    * The Technical Developer Community (TDC) is open to any developer, end user or subject matter expert that chooses to participate in the activities of OAI, regardless of whether the participant is employed by an OAI Member company. Influence on the evolution of the OAS comes by engaging with the TDC in technical discussions and by contributing to the project. [[Read more here](https://www.openapis.org/participate/how-to-contribute/governance#TDC)]
+    * The Technical Steering Committee (TSC) is open to any developer, end user or subject matter expert that chooses to participate in the activities of OAI, regardless of whether the participant is employed by an OAI Member company. Influence on the evolution of the OAS comes by engaging with the TDC in technical discussions and by contributing to the project. [[Read more here](https://www.openapis.org/participate/how-to-contribute/governance#TDC)] 
```

Comment:
Might need to re-word as it could be interpreted that membership in TSC is by participation. Also, there's a TDC references in this line that needs updating.